src/components/ColorInput.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ColorContainer = styled.div`
+  display: flex;
+`;
+
+const Color = styled.div`
+  height: 100%;
+  width: 50px;
+  background-color: ${props => props.color};
+  border: ${props => props.color};
+`;
+
+const StyledColorInput = styled.input`
+  display: flex;
+  background-color: ${props => props.color};
+  border: ${props => props.color};
+  opacity: 0.8;
+  height: 20px;
+  width: 20px;
+`;
+
+export default function ColorInput({ userGroup }) {
+  function getAvailableColors(userGroup) {
```

Comment:
i think here and in line 37 you dont need `userGroup` as parameter. it is globally available in the function `ColorInput ` because you destructure it above. so you can write `function getAvailableColors() {...}` here and `getAvailableColors().map`